server/controllers/journal/fiscal.js

```diff
@@ -50,11 +51,11 @@ function close(id, user_id, data, cb) {
   function init() {
     cfg.user_id = user_id;
     cfg.project_id = 1; // HBB by default
-    transactionDate = cfg.isForClosing ? forcingDate : new Date();
-    return [
+    transactionDate = cfg.isForClosing ? util.toMysqlDate(forcingDate) : util.toMysqlDate();
+    return q.when([
```

Comment:
I needed a promise because after the init() function there is a ".spread(...) ". The error was that the init() function returns a simple array instead of a promise or if it was in a promise chain